node/src/chain_spec/turing.rs

```diff
@@ -69,13 +69,15 @@ pub fn turing_development_config() -> ChainSpec {
 						get_collator_keys_from_seed("Bob"),
 					),
 				],
-				get_account_id_from_seed::<sr25519::Public>("Alice"),
 				endowed_accounts,
 				REGISTERED_PARA_ID.into(),
 				vec![],
 				vec![],
 				vec![get_account_id_from_seed::<sr25519::Public>("Alice")],
```

Comment:
seems like you added extra person to technical council. i'm assuming for testing voting. do you need to add a second person to general council as well?

Reply:
I only needed to test the technical council